packages/styles/scss/patterns/card-section/_card-section.scss

```diff
@@ -50,96 +36,61 @@
     &__title {
       @include type-style('expressive-heading-02');
 
+      padding-left: $carbon--spacing-05;
       padding-bottom: $carbon--spacing-05;
     }
     &__cards {
-      display: flex;
-      float: right;
-      flex-wrap: wrap;
-    }
-    .bx--card-link {
-      @include carbon--make-col(8, 8);
+      &__container {
+        @include carbon--make-container;
+      }
+      &__row {
+        @include carbon--make-row;
+      }
+      &__col {
+        @include carbon--make-col-ready;
+        @include carbon--make-col(4, 4);
```

Comment:
When we added this line earlier I wasn't really thinking. All it does is set 100%. Since it's 100% be default I don't think it's needed.